graviti/utility/file.py

```diff
@@ -25,16 +76,61 @@ class File(RemoteFileMixin):
     __slots__ = ("url", "_checksum")
 
     def __init__(self, url: str, checksum: str) -> None:
-        super().__init__("")
         self.url = READ_ONLY_URL(url)
         self._checksum = checksum
 
     def _repr_head(self) -> str:
-        return f'{self.__class__.__name__}("{shorten(self._checksum)}")'
+        return f'{self.__class__.__name__}("{self._checksum[:7]}")'
```

Comment:
why stoping using `shorten`?

Reply:
the goal of `shorten` is easy to achieve, why should we use that simple function...